core/commands/add.go

```diff
@@ -36,6 +36,26 @@ Adds contents of <path> to ipfs. Use -r to add directories.
 Note that directories are added recursively, to form the ipfs
 MerkleDAG.
 `,
+		LongDescription: `
+Adds contents of <path> to ipfs. Use -r to add directories.
+Note that directories are added recursively, to form the ipfs
+MerkleDAG.
+
+Note that using the wrap option, '-w', wraps the file (or files,
```

Comment:
'Also note...'  maybe? It sounds weird that we say "note that" to start two sentences in a row